### DISCUSSION TITLE
Getting the foreground color of text as shown by the Editor Tokens and Scopes Inspector

### DISCUSSION THREAD

**Discussion by BWotka** [microsoft/vscode-discussions#133] [Extension Development QnA] [answered]
I'm working to update an Extension which was left of by its original creator 5 years ago.
Some of the features used seem no longer available.
Is there still a way to get the editor color for a type?
I did not find anything in my search in the API.

I'm looking for something like this:
`workspaceConfig.get('editor.comment.foreground') `
To return the foreground color of a comment in the current editor.
Any help or hint at the API would be greatly appreciated.

**Comment by BWotka** [comment]
I figured some things out, this apparently never worked. The extension has always used a workaround.
This would mean this is more of a feature request instead of a question.
Is it possible to mark this as a feature request? Or should I submit an issue in the original vscode repo?

**Comment by gjsjohnmurray** [comment, marked as answer]
Probably already covered by https://github.com/microsoft/vscode/issues/32813
